src/components/GagesBarChartAnimation.vue

```diff
@@ -3010,6 +4435,41 @@ $axis: rgb(100,100,100);
     display: block;
     text-align: center;
   }
+
+  #infographic  {
+    width: 100%;
+
+    .text-big {
+      font-size: 26px;
+      color: black;
+    }
+    .text-small {
+      font-size: 20px;
+      color: #686867;
+    }
+    .water-background {
+      fill:#d7e7f2;
+    }
+    .svg-motion {
+      fill:none;
+      stroke:#e0d6d3;
+      stroke-width:3;
+    }
+    .nwis-variables  {
+      text-anchor: end;
+      font-size: 20px;
+      fill: #686867;
+      
+    }
+
+    .nwis-data  {
+      text-anchor: start;
+      font-size: 20px;
+      fill: grey;
+      
+    }
+  }
+
   @media screen and (min-width: 600px){
```

Comment:
I think this block is not used.

Reply:
these are for the text align in the infographic

Reply:
Sorry, my highlighting was off, it was the @media block. No big deal though.